.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,47 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  contents: read
+
+# PR updates can happen in quick succession leading to this
+# workflow being trigger a number of times. This limits it
+# to one run per PR.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  main:
+    permissions:
+      pull-requests: read  # for amannn/action-semantic-pull-request to analyze PRs
+      statuses: write  # for amannn/action-semantic-pull-request to mark status of analyzed PR
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb # v5.0.2
+        with:
+          types: |
+            feat
+            fix
+            docs
+            test
+            ci
+            chore
+            [Bot] docs
+          scopes: |
+            controller
+            applicationset
+            repo-server
+            api
+            ui
+            deps
+            cli
+          requireScope: false
```

Comment:
We are not using scopes and I don't remember having this defined anywhere for the project. Unless we agree on using scope and document it somewhere, I think we should just remove it and keep it simple.

Reply:
Folks have used scopes in the past, voluntarily. I added a few that I've seen before.

I think this file is sufficient documentation of the available scopes.

They are already marked as optional: `requireScope: false`.

Reply:
Scopes previously used:

```
$ git log --pretty="format:%s %ae" | egrep -i '^[a-z]+\(' | sed -r 's|[a-z]+\\(||' | sed -r 's|\).*||' | sort | uniq
app cmd
application-tiles.tsx
applications-table.tsx
applications-tiles.tsx
applicationset
bug_report
cli
deps
deps-dev
dex
docs
fix
gitpod
health
helm
high availability
info.go
jwt
logging.go
manifests
prom
repository.go
revision.tsx
ui
user-guide
users
```

Reply:
I understand. I just don't think we should enforce a predefined list for scopes. If someone wants to create a PR with the following title, it should be fine imo:
`fix (jwt): verify aud correctly`

Reply:
That's fair, I'll yeet the list